reobin/vimcolorschemes
# feat: Add search input

## Type of PR


## Description

Adds a search feature.

1. Search index is uploaded to Elasticsearch at build time using elasticjs
2. Search index is queried using a node proxy API

## Related issue

Closes #74 

## QA Instructions

1. Enter search query
2. Check search results
3. Check out the use of local storage to keep the search input when returning from a repository

## Added unit tests?


## Added documentation?
